docs/references/protocol/transactions/types/oracleset.md

```diff
@@ -0,0 +1,97 @@
+---
+html: OracleSet.html 
+parent: transaction-types.html
+blurb: Create or update an existing price oracle.
+labels:
+  - Oracle
+status: not_enabled
+---
+# OracleSet
+_(Requires the [PriceOracle amendment][] {% not-enabled /%})_
+
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/ripple/app/tx/impl/SetOracle.cpp "Source")
+
+Creates a new `Oracle` ledger entry or updates the fields of an existing one, using the Oracle ID.
+
+The oracle provider must complete these steps before submitting this transaction:
+
+1. Create or own the XRPL account in the `Owner` field and have enough XRP to meet the reserve and transaction fee requirements.
+2. Publish the XRPL account public key, so it can be used for verification by dApps.
+3. Publish a registry of available price oracles with their unique `OracleDocumentID`.
+
+
+## Example OracleSet JSON
+
+```json
+{
+  "TransactionType": "OracleSet",
+  "Account": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+  "OracleDocumentID": 34,
+  "Provider": "70726F7669646572",
+  "LastUpdateTime": 743609014,
+  "AssetClass": "63757272656E6379",
+  "PriceDataSeries": [
+    {
+      "PriceData": {
+        "BaseAsset": "XRP",
+        "QuoteAsset": "USD",
+        "AssetPrice": 740,
+        "Scale": 3
+      }
+    }
+  ]
+}
+```
+
+
+## OracleSet Fields
+
+| Field              | JSON Type | Internal Type | Required? | Description |
+|--------------------|-----------|---------------|-----------|-------------|
+| `Account`          | String    | AccountID     | Yes       | This account must match the account in the `Owner` field of the `Oracle` object. |
+| `OracleDocumentID` | Number    | UInt32        | Yes       | A unique identifier of the price oracle for the `Account`. |
+| `Provider`         | String    | Blob          | Variable  | An arbitrary value that identifies an oracle provider, such as Chainlink, Band, or DIA. This field is a string, up to 256 ASCII hex encoded characters (0x20-0x7E). This field is required when creating a new `Oracle` ledger entry, but is optional for updates. |
+| `URI`              | String    | Blob          | No        | An optional Universal Resource Identifier to reference price data off-chain. This field is limited to 256 bytes. |
+| `LastUpdateTime`   | Number    | UInt32        | Yes       | The time the data was last updated, represented in the [ripple epoch time](https://xrpl.org/basic-data-types.html#specifying-time). |
```

Comment:
Use a common link here too.

```suggestion
| `LastUpdateTime`   | Number    | UInt32        | Yes       | The time the data was last updated, in [seconds since the Ripple Epoch][]. |
```

Also, this contradicts the definition in the ledger entry page that says it's Unix time. (And the example once again represents a date in 2023, so I suggest updating that.)